aten/src/ATen/native/native_functions.yaml

```diff
@@ -3220,6 +3220,10 @@
     MkldnnCPU: mkldnn_relu_
     QuantizedCPU: relu_quantized_cpu_
 
+- func: relu6(Tensor self) -> Tensor
+
+- func: relu6_(Tensor(a!) self) -> Tensor(a!)
+
```

Comment:
We should add `python_module: nn` annotations here.

relu6 should not be in the torch.* namespace, instead, it should be in the torch.nn.functional namespace. Adding `python_module: nn` makes it so that relu6 is bound to `torch._C._nn.relu6`. "relu" is in the torch.* namespace but I think that was an oversight.